packages/expo-updates/ios/EXUpdates/AppLauncher/AppLauncherNoDatabase.swift

```diff
@@ -23,27 +23,12 @@ public final class AppLauncherNoDatabase: NSObject, AppLauncher {
 
   public func launchUpdate(withConfig config: UpdatesConfig) {
     launchedUpdate = EmbeddedAppLoader.embeddedManifest(withConfig: config, database: nil)
-    if let launchedUpdate = launchedUpdate {
-      if launchedUpdate.status == UpdateStatus.StatusEmbedded {
-        precondition(assetFilesMap == nil, "assetFilesMap should be null for embedded updates")
-        launchAssetUrl = Bundle.main.url(
-          forResource: EmbeddedAppLoader.EXUpdatesBareEmbeddedBundleFilename,
-          withExtension: EmbeddedAppLoader.EXUpdatesBareEmbeddedBundleFileType
-        )
-      } else {
-        launchAssetUrl = Bundle.main.url(
-          forResource: EmbeddedAppLoader.EXUpdatesEmbeddedBundleFilename,
-          withExtension: EmbeddedAppLoader.EXUpdatesEmbeddedBundleFileType
-        )
-
-        var assetFilesMapLocal: [String: String] = [:]
-        for asset in launchedUpdate.assets()! {
-          if let assetKey = asset.key, let localUrl = UpdatesUtils.url(forBundledAsset: asset) {
```

Comment:
Just grepping, I think this might be the only caller of UpdatesUtils.url()

Reply:
That method will still be needed for my asset exclusion changes